comms/iio-sensor-proxy/Makefile

```
@@ -0,0 +1,25 @@
+PORTNAME=	iio-sensor-proxy
+DISTVERSION=	3.6
+CATEGORIES=	comms
+
+MAINTAINTER=	dhs@frame.work
+COMMENT=	Proxies sensor devices to applications through D-BUS
+WWW=		https://gitlab.freedesktop.org/hadess/iio-sensor-proxy
+
+USES=		pkgconfig meson
+USE_GNOME+=	glib20
+USE_GITLAB=	yes
+GL_SITE=	https://gitlab.freedesktop.org
+GL_ACCOUNT=	hadess
+
+LICENSE=	GPLv3
+LICENSE_FILE=	${WRKSRC}/COPYING
+
+LIB_DEPENDS+=	libudev.so:devel/libudev-devd \
```

```suggestion
BUILD_DEPENDS=	evdev-proto>0:devel/evdev-proto
LIB_DEPENDS+=	libudev.so:devel/libudev-devd \
```

@lwhsu I think this should fix the poudriere build